pkg/validate/container.go

```diff
@@ -109,6 +112,39 @@ var _ = framework.KubeDescribe("Container", func() {
 			testExecSync(rc, containerID)
 		})
 	})
+
+	Context("runtime should support adding volume and device [Conformance]", func() {
+		var podID string
+		var podConfig *runtimeapi.PodSandboxConfig
+
+		BeforeEach(func() {
+			podID, podConfig = createPodSandboxForContainer(rc)
+			pullImageList(ic, []string{defaultContainerImage})
```

Comment:
We don't need pull the image here before creating container, since `createContainer()` will pull the image.